### DISCUSSION TITLE
T.38 support?

### DISCUSSION THREAD

**Discussion by ArtisWorld** [sipsorcery-org/sipsorcery#763] [Q&A]
Does sipsorcery support T.38 for SIP fax?

Thanks, Arti.

**Comment by sipsorcery** [comment]
No support for T38 or anything fax related.

Been a long time since I've come across someone asking about fax. I do recall the recall the days of setting up Asterisk + Hylafax but thankfully they're now a dim memory.

**Reply by ArtisWorld** [reply]
ok, thanks anyway.
In some professions (lawyers, doctors, etc.) faxes are still used extensively.
